tests/unit/commands/validate/test_cli.py

```diff
@@ -106,16 +106,6 @@ def test_lint_invalid_region_argument_fails(self, click_patch, matches_patch, ar
         with self.assertRaises(UserException):
             _lint(ctx=ctx_lint_mock(debug=False, region="region"), template=template_path)
 
-    @patch("botocore.utils.validate_region_name")
```

Comment:
Why we have removed this test, can we add it back with more generic solution that you have implemented?